shell/browser/api/atom_api_top_level_window.cc

```diff
@@ -555,8 +555,10 @@ std::vector<int> TopLevelWindow::GetPosition() {
 }
 void TopLevelWindow::MoveAbove(const std::string& sourceId,
                                mate::Arguments* args) {
-  if (!window_->MoveAbove(sourceId))
-    args->ThrowError("Invalid media source id");
+  #if  BUILDFLAG(ENABLE_DESKTOP_CAPTURER)
+    if (!window_->MoveAbove(sourceId))
+      args->ThrowError("Invalid media source id");
+  #endif
```

Comment:
Maybe throw error in the other case instead.

```
#else
args->ThrowError("enable_desktop_capturer=true to use this feature");
#endif
```

Reply:
Lint is [unhappy](https://circleci.com/gh/electron/electron/303026?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) about this indentation